mm2src/mm2_main/src/lp_swap/komodefi.swap_v2.pb.rs

```diff
@@ -0,0 +1,114 @@
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct SignedMessage {
+    #[prost(bytes="vec", tag="1")]
+    pub from: ::prost::alloc::vec::Vec<u8>,
+    #[prost(bytes="vec", tag="2")]
+    pub signature: ::prost::alloc::vec::Vec<u8>,
+    #[prost(bytes="vec", tag="3")]
+    pub payload: ::prost::alloc::vec::Vec<u8>,
+}
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct MakerNegotiation {
+    #[prost(uint64, tag="1")]
+    pub started_at: u64,
+    #[prost(uint64, tag="2")]
+    pub payment_locktime: u64,
+    #[prost(bytes="vec", tag="3")]
+    pub secret_hash: ::prost::alloc::vec::Vec<u8>,
+    #[prost(bytes="vec", tag="4")]
+    pub maker_coin_htlc_pub: ::prost::alloc::vec::Vec<u8>,
```

Comment:
There are multiple usage of `vec<u8>` here, we should use more stricter types for over the wire messages but I understand that for protobuf fixed length arrays are not supported to provide the backward compatibility features that make protobuf special. Should we be concerned about the usage of vec<u8> here? I guess validation of messages will remove any concerns related to the usage of vecs. Again, just opening this for discussion :)

Reply:
Pubkeys and secret hash may have variable length, e.g. 20 or 32 bytes for secret hash (and maybe we will use hashes with a longer digest in the future). So, having `Vec<u8>` actually gives us more flexibility and reduces the chances of rendering incompatibility issues with hacky workarounds 🙂